pkg/reconciler/common/transformers.go

```diff
@@ -785,19 +785,16 @@ func CopyConfigMapWithForceUpdate(configMapName string, expectedValues map[strin
 		if err != nil {
 			return err
 		}
```

Comment:
@pratap0007 add a check to verify the size of the `expectedValues` map. If the size is zero, just return. no action needed